lib/gscraper/account/users.ex

```diff
@@ -4,9 +4,50 @@ defmodule Gscraper.Account.Users do
   """
 
   import Ecto.Query, warn: false
+
+  alias Gscraper.Account.Passwords
+  alias Gscraper.Account.Queries.UserQuery
   alias Gscraper.Account.Schemas.User
   alias Gscraper.Repo
 
+  @doc """
+  Gets a single user.
+
+  Raises `Ecto.NoResultsError` if the User does not exist.
+
+  ## Examples
+
+      iex> find_by_id!(123)
+      %User{}
+      iex> find_by_id!(456)
+      ** (Ecto.NoResultsError)
+
+  """
+  def find_by_id!(user_id) do
+    user_id
+    |> UserQuery.find_by_id()
+    |> Repo.one!()
```

Comment:
Not sure, what do you think about using `Repo.get!(id)` here and `Repo.get_by!(username: username)`, so we don't have to do our own queries. 🤔

Reply:
yes, it's way more concise. Fixed in 4439b20.